contract/fuel-toolchain.toml

```
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "beta-4-rc.2"
\ No newline at end of file
+channel = "calldelegation_beta_5"
```

```suggestion
channel = "nightly-2023-12-06"
```
At least until beta-5 channel is released